ncm-cron/src/main/pan/components/cron/schema.pan

```diff
@@ -73,24 +156,26 @@ type structure_cron = {
       lists or ranges.  Time specifications such as ranges, lists and steps are
       supported except for named values (e.g. "1" must be used instead of "mon").}
     'timing' ? structure_cron_timing
-    @{Command line to execute, including all its options.}
+    @documentation{Command line to execute, including all its options.}
     'command' : string
-    @{An optional comment to add at the beginning of the cron file.}
+    @documentation{An optional comment to add at the beginning of the cron file.}
     'comment' ? string
-    @{An optional dict containing environment variable that must be
+    @documentation{An optional dict containing environment variable that must be
       defined before executing the command. Key is
       the variable name, value is variable value.}
     'env' ? string{}
     'log' ? structure_cron_log
     'syslog' ? structure_cron_syslog
 } with {
     if (exists(SELF['log']) && exists(SELF['syslog'])) {
-        error("At most one of log or syslog can be defined");
+        error("Only one of log or syslog may be defined, not both.");
+    };
+    if (exists(SELF['timing']) && exists(SELF['frequency'])) {
```

Comment:
i have seen constructs like this in other schemas as well, maybe we should add function to pan to test for this, eg
`oneof(SELF, 'timing', 'frequency')` (or above `maxone(SELF, 'log', 'syslog')`). obviously these shouldn't call `error`, but this is a lot of boilerplate imho.

Reply:
I agree, but this probably indicates that the schema is badly designed.